llvm/lib/CheerpUtils/Utility.cpp

```diff
@@ -1387,4 +1387,55 @@ unsigned getVectorBitwidth(const FixedVectorType* vecType)
 	return vectorBitwidth;
 }
 
+bool hasSIMDAttribute(const Function* F)
```

Comment:
Here in these 2 functions, you could try to work on the StringRef, without creating new temporary std::strings if possible

Reply:
I've also changed the feature strings to be StringRefs. The join function is fine with taking a vector of StringRefs